packages-exp/app-exp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@firebase/app-exp",
-  "version": "0.800.0",
+  "version": "0.0.800",
```

Comment:
8 is the next major firebase version. 800 makes it not conflict with any existing version. It can be any version number actually because we will append a hash string to it when publishing to npm, e.g. `0.0.800-hashvalue`. This is just what I chose to use.